docs/UserGuide.md

```diff
@@ -153,6 +229,16 @@ Example of usage:
 
 Note: You can find the task indexes with `list -t`  
 
+Example of output:
+```
+Command: 
+⋗	edit -t 1 buy stuff
+════════════════════════════════════════════════════════════════════════════════
+Your task has been edited successfully.
+
+════════════════════════════════════════════════════════════════════════════════
+```
+
```

Comment:
Maybe show the output of
1. `list -t`
2. `edit -t 1 buy stuff`
3. `list -t`
Or it would be wise to show the new edited task in the output rather than just `Your task has been edited successfully.`